RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -882,6 +885,7 @@ string freeRunCombatString(monster enemy, location loc, boolean inCombat)
 		// TODO add fam weight buffing
 		// boots and bander share same counter
 		int banderRunsLeft = floor((familiar_weight($familiar[Pair of Stomping Boots]) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int();
+		if(in_wereprof() && !is_werewolf()) return "";
```

Comment:
can this be removed? Already checked on line 825

Reply:
See other comment